$NetBSD: patch-ab,v 1.4 1999/12/02 15:50:45 frueauf Exp $

Make this work on NetBSD.
For Solaris, make the DATADIRNAME share.

--- configure.orig	Tue Sep  7 17:12:37 1999
+++ configure	Thu Dec  2 16:03:10 1999
@@ -2505,7 +2505,7 @@
             CFLAGS_RDYNAMIC="-Wl,-E"
             ;;
 
-          openbsd* | freebsd2* | osf4* | irix* )
+          openbsd* | freebsd2* | osf4* | irix* | netbsd* )
             CFLAGS_RDYNAMIC=""
             ;;
 
@@ -2553,6 +2553,9 @@
         NAME_OF_DEV_URANDOM="/dev/urandom"
         DYNLINK_MOD_CFLAGS="-shared -rdynamic -fpic -Wl,-Bshareable -Wl,-x"
         ;;
+    *-netbsd*)
+	DYNLINK_MOD_CFLAGS="-shared -rdynamic -fpic -Wl,-Bshareable -Wl,-x"
+	;;
     *)
         NAME_OF_DEV_RANDOM="/dev/random"
         NAME_OF_DEV_URANDOM="/dev/urandom"
@@ -3921,7 +3924,7 @@
   cat conftest.$ac_ext >&5
   rm -rf conftest*
   CATOBJEXT=.mo
-		   DATADIRNAME=lib
+		   DATADIRNAME=share
 fi
 rm -f conftest*
 		INSTOBJEXT=.mo
